UnityAds/src/main/java/com/mopub/mobileads/UnityInterstitial.java

```diff
@@ -99,8 +100,9 @@ public void onUnityAdsAdLoaded(String placementId) {
         }
 
         @Override
-        public void onUnityAdsFailedToLoad(String placementId) {
-            MoPubLog.log(CUSTOM, ADAPTER_NAME, "Unity interstitial failed to load for placement " + placementId);
+        public void onUnityAdsFailedToLoad(String placementId, UnityAdsLoadError error, String message) {
+            MoPubLog.log(CUSTOM, ADAPTER_NAME, "Unity interstitial failed to load for placement " +
+                placementId + ", with error message: " + message);
             MoPubLog.log(LOAD_FAILED, ADAPTER_NAME, MoPubErrorCode.NETWORK_NO_FILL.getIntCode(), MoPubErrorCode.NETWORK_NO_FILL);
```

Comment:
Should we map to an appropriate error if it exists rather than just `NO_FILL`?

Reply:
+1 here. Shall we tackle this now on with some later Adapter refactoring?